patches/lua-tree.patch

```diff
@@ -701,6 +830,28 @@ index 6c92098cc..c3c3c12fe 100644
  local get_mime_type
  local get_error_template
  do
+@@ -1454,16 +1257,12 @@ do
+   end
+ end
+ 
+-local time_ns
+-do
+-  local nanop = ffi_new("nanotime[1]")
+-  function time_ns()
+-    -- CLOCK_REALTIME -> 0
+-    C.clock_gettime(0, nanop)
+-    local t = nanop[0]
++-- No reason to yield a Lua coroutine based on `ngx.get_phase()`,
++-- as we're not running this within Nginx anyway.
++_M.yield = function() return end
+ 
+-    return tonumber(t.tv_sec) * 1e9 + tonumber(t.tv_nsec)
+-  end
++local time_ns = function()
++    return require("socket").gettime() * 10^9
+ end
+ _M.time_ns = time_ns
```

Comment:
Please document this in the patch description.

Reply:
Will defer to [this comment](https://github.com/Kong/goks/pull/27#discussion_r957855744).

Reply:
I mentioned this earlier, but I think we can simply remove `time_ns()` from util.